### ISSUE TITLE
Render mesh depth and save caches 这个步骤错误

### ISSUE THREAD

**Issue by chenhui2016** [Visionary-Laboratory/Proxy-GS#6]
十分感谢您开源此项目，恭喜中2026cvpr oral(#^.^#)， 我在运行Render mesh depth and save caches 这个步骤时报错，请问是什么原因？
`󰕈 ➜  cd /home/ch/sn_d/code/3dgs/dynamic/Proxy-GS ; /usr/bin/env /home/ch/anaconda3/envs/proxygs/bin/python /home/ch/.vscode/extensions/ms-python.debugpy-2026.6.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher 39819 -- /home/ch/sn_d/code/3dgs/dynamic/Proxy-GS/mesh_render.py -s /home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/pose_block/block_5 -m /home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/output/block_5 -i /home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/train/small_city_road_horizon --ply_mesh /home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/mesh/tsdf_fusion_post_block5.ply --depth_npy_dir /home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/depth 
0
No laspy
Setting up [LPIPS] perceptual loss: trunk [vgg], v[0.1], spatial [off]
/home/ch/anaconda3/envs/proxygs/lib/python3.9/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/home/ch/anaconda3/envs/proxygs/lib/python3.9/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG16_Weights.IMAGENET1K_V1`. You can also use `weights=VGG16_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Loading model from: /home/ch/anaconda3/envs/proxygs/lib/python3.9/site-packages/lpips/weights/v0.1/vgg.pth
found tf board
2026-06-17 19:34:29,775 - INFO: args: Namespace(feat_dim=32, n_offsets=10, fork=2, use_feat_bank=False, source_path='/home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/pose_block/block_5', model_path='/home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/output/block_5', images='/home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/train/small_city_road_horizon', resolution=-1, white_background=False, random_background=False, resolution_scales=[1.0], data_device='cpu', eval=False, ds=1, ratio=1, undistorted=False, appearance_dim=0, add_opacity_dist=False, add_cov_dist=False, add_color_dist=False, add_level=False, extend=1.1, dist2level='round', base_layer=10, visible_threshold=0.0, update_ratio=0.2, progressive=True, dist_ratio=0.999, levels=-1, init_level=-1, extra_ratio=0.25, extra_up=0.01, iterations=100000, position_lr_init=0.0, position_lr_final=0.0, position_lr_delay_mult=0.01, position_lr_max_steps=50000, offset_lr_init=0.01, offset_lr_final=0.0001, offset_lr_delay_mult=0.01, offset_lr_max_steps=50000, feature_lr=0.0075, opacity_lr=0.02, scaling_lr=0.007, rotation_lr=0.002, mlp_opacity_lr_init=0.002, mlp_opacity_lr_final=2e-05, mlp_opacity_lr_delay_mult=0.01, mlp_opacity_lr_max_steps=50000, mlp_cov_lr_init=0.004, mlp_cov_lr_final=0.004, mlp_cov_lr_delay_mult=0.01, mlp_cov_lr_max_steps=50000, mlp_color_lr_init=0.008, mlp_color_lr_final=5e-05, mlp_color_lr_delay_mult=0.01, mlp_color_lr_max_steps=50000, mlp_featurebank_lr_init=0.01, mlp_featurebank_lr_final=1e-05, mlp_featurebank_lr_delay_mult=0.01, mlp_featurebank_lr_max_steps=50000, appearance_lr_init=0.05, appearance_lr_final=0.0005, appearance_lr_delay_mult=0.01, appearance_lr_max_steps=50000, percent_dense=0.01, lambda_dssim=0.2, start_stat=500, update_from=1500, coarse_iter=10000, coarse_factor=1.5, update_interval=100, update_until=40000, update_anchor=True, min_opacity=0.005, success_threshold=0.8, densify_grad_threshold=0.0002, compute_cov3D_python=False, debug=False, ip='127.0.0.1', port=6009, debug_from=-1, detect_anomaly=False, warmup=False, use_wandb=False, test_iterations=[-1], save_iterations=[30000, 50000], quiet=False, checkpoint_iterations=[], start_checkpoint=None, gpu='-1', ply_path=None, ply_mesh='/home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/mesh/tsdf_fusion_post_block5.ply', depth_npy_dir='/home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/depth')
[10000, 20000, 30000, 40000, 50000, 60000, 70000, 80000, 90000, 100000]
[30000, 50000, 100000]
2026-06-17 19:34:29,775 - INFO: Optimizing /home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/output/block_5
Output folder: /home/ch/gm_d/Linux_data/datasets/3dgs/matrixcity/small_city/stress/output/block_5 [17/06 19:34:29]
Reading Training Transforms [17/06 19:34:29]
Loading dataset: 100%|█████████████████████████████████████| 1580/1580 [01:07<00:00, 23.33it/s, num=1570/1580]
Reading Test Transforms [17/06 19:35:37]
Loading dataset: 190it [00:07, 24.77it/s, num=180/185]                                                        
Loading Training Cameras [17/06 19:35:46]
Loading cameras: 100%|███████████████████████████████████████████████████| 1765/1765 [00:04<00:00, 396.43it/s]
Loading Test Cameras [17/06 19:35:50]
Loading cameras: 0it [00:00, ?it/s]
2026-06-17 19:35:50,672 - INFO: Using black background
Building octree time: 0 min 0.002568483352661133 sec [17/06 19:35:51]
Branches of Tree: 2 [17/06 19:35:51]
Base Layer of Tree: 10 [17/06 19:35:51]
Visible Threshold: 0.0 [17/06 19:35:51]
Appearance Embedding Dimension: 0 [17/06 19:35:51]
LOD Levels: 8 [17/06 19:35:51]
Initial Levels: 4 [17/06 19:35:51]
Initial Voxel Number: 1063721 [17/06 19:35:51]
Min Voxel Size: 0.0001425857626600191 [17/06 19:35:51]
Max Voxel Size: 0.018250977620482445 [17/06 19:35:51]
2026-06-17 19:35:51,796 - INFO: Branches of Tree: 2
2026-06-17 19:35:51,796 - INFO: Base Layer of Tree: 10
2026-06-17 19:35:51,796 - INFO: Visible Threshold: 0.0
2026-06-17 19:35:51,796 - INFO: Appearance Embedding Dimension: 0
2026-06-17 19:35:51,796 - INFO: LOD Levels: 8
2026-06-17 19:35:51,796 - INFO: Initial Levels: 4
2026-06-17 19:35:51,796 - INFO: Initial Voxel Number: 1063721
2026-06-17 19:35:51,796 - INFO: Min Voxel Size: 0.0001425857626600191
2026-06-17 19:35:51,796 - INFO: Max Voxel Size: 0.018250977620482445
/home/ch/anaconda3/envs/proxygs/lib/python3.9/site-packages/torch/utils/cpp_extension.py:2356: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation. 
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
  warnings.warn(
[chunk 1/1] no hits [17/06 19:36:02]
[chunk 1/1] no hits [17/06 19:36:02]`
